src/util/geniso

```diff
@@ -120,7 +120,7 @@ case "${LEGACY}" in
 		;;
 	0)
 		# copy isolinux bootloader
-		cp ${ISOLINUX_BIN} ${dir}
+		install -m 644 ${ISOLINUX_BIN} ${dir}
```

Comment:
Merged (in the replacement `util/genfsimg`) as https://github.com/ipxe/ipxe/commit/25b675c3d